ts/src/pro/bitfinex.ts

```diff
@@ -352,13 +352,13 @@ export default class bitfinex extends bitfinexRest {
             const orderbook = this.orderbooks[symbol];
             if (isRaw) {
                 const id = this.safeString (message, 1);
-                const price = this.safeFloat (message, 2);
+                const price = this.safeString (message, 2);
                 const size = (message[3] < 0) ? -message[3] : message[3];
                 const side = (message[3] < 0) ? 'asks' : 'bids';
                 const bookside = orderbook[side];
                 // price = 0 means that you have to remove the order from your book
-                const amount = (price > 0) ? size : 0;
-                bookside.store (price, amount, id);
+                const amount = Precise.stringGt (price, '0') ? size : '0';
```

Comment:
@carlosmiei On the other PR you wrote

> First we uniformize price but not amount, second I'm not sure our orderbooks are ready to handle strings if the user changes the number's type 🤔

and I'm not sure what you meant

Reply:
@samgermain regarding the "uniformize" part, I don't remember either, but I'm still not sure the orderbook internally works well with strings (probably not), so if the user changes the number type, we might have a problem. Nonetheless, I think that is a different topic